cmake/schemes/url_sha1_boost.cmake.in

```diff
@@ -88,8 +89,9 @@ else()
 endif()
 
 if("@MSVC@" OR "@MINGW@")
```

Comment:
Does this trick needed for Visual Studio? I think default installation of Visual Studio is in `Program Files` (i.e. has space) and works fine(?) Have you experienced any problems?